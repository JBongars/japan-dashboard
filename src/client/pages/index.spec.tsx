import * as React from "react";
import * as RootPage from ".";
import { shallow, ShallowWrapper } from "enzyme";

describe("# Root Page", (): void => {
  it("should render a root page", (): void => {
    const wrapper: ShallowWrapper = shallow(<RootPage.default />);

    expect(wrapper).toMatchSnapshot();
  });
});
