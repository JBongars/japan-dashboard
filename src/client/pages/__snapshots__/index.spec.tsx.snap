// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`# Root Page should render a root page 1`] = `ShallowWrapper {}`;
